docs/reference/mpremote.rst

```diff
@@ -238,9 +238,17 @@ The full list of supported commands are:
   The ``cp`` command uses a convention where a leading ``:`` represents a remote
   path. Without a leading ``:`` means a local path. This is based on the
   convention used by the `Secure Copy Protocol (scp) client
-  <https://en.wikipedia.org/wiki/Secure_copy_protocol>`_. All other commands
-  implicitly assume the path is a remote path, but the ``:`` can be optionally
-  used for clarity.
+  <https://en.wikipedia.org/wiki/Secure_copy_protocol>`_. 
+  
+  The ``rm -r`` command accepts ``:`` to refer to the current remote path to allow a 
+  directory tree to be removed from the default path of the device (e.g. 
+  ``/flash`` or ``/``).
```

Comment:
I think it would be good to explicitly have `mpremote rm -r :` as an example here.

Reply:
Added 2 exaples, one to remove a reltive folder `libs`, and one to clear out the entire device.